### DISCUSSION TITLE
Pin The Dashboard Feature

### DISCUSSION THREAD

**Discussion by pankajsoni22** [apache/superset#25840] [Ideas]
### Pin Feature
I have 150 dashboards on superset and always face issue to search because to search I need to remember something from dashboard title.

It will be good if we can have pin feature so that pinned dashboards stay at the top.

This can be a feature flag enabled feature 
`PINNING=True`

**Comment by drummerwolli** [comment]
there is a "favourite" button, and you can search for your favourite dashboards. my standard browser bookmark is actually to go to the list of favourite dashboards: https://your.superset.net/dashboard/list/?filters=(favorite:(label:Yes,value:!t))

![image](https://github.com/apache/superset/assets/3867569/67fc1635-b876-40a1-8ea0-1467bc17bfd8)

so i think this functionality is already there

**Reply by pankajsoni22** [reply]
We have more than 45 favourites. For consumers it is not easy to search for them. I want to provide a better dashboard experience to user where they can see their own dashboards at the top.

It becomes easy if we can pin the important dashboards.
